apps/communications/dialer/test/unit/call_log_test.js

```diff
@@ -886,29 +896,47 @@ suite('dialer/call_log', function() {
 
         CallLog.render();
 
-        this.sinon.stub(Contacts, 'findByNumber');
+        allLogs = document.body.getElementsByClassName('log-item');
+      });
 
-        var contactEvent = {
-          reason: 'remove',
-          contactID: incomingGroup.contact.id
-        };
-        navigator.mozContacts.oncontactchange(contactEvent);
+      suite('no matching contact id found', function() {
+        setup(function() {
+          var contactEvent = {
+            reason: 'remove',
+            contactID: '789'
+          };
+          this.sinon.spy(CallLogDBManager, 'removeGroupContactInfo');
```

Comment:
We should call ```this.sinon.stub()``` instead of ```this.sinon.spy()``` here since we're stubbing the function, not spying on calls to it. It states intent more clearly.